src/App.tsx

```diff
@@ -346,6 +346,9 @@ const Scenes: NavigationRouteConfigMap = {
     FeedInfo: {
         screen: FeedInfoContainer,
     },
+    WebFeedInfo: {
```

Comment:
I would either call this `PublicFeedInfo` or maybe `RSSFeedInfo`

Reply:
resolved in 272386a